manifests/klusterlet/management/klusterlet-agent-deployment.yaml

```diff
@@ -118,10 +118,12 @@ spec:
             scheme: HTTPS
             port: 8443
           initialDelaySeconds: 2
+        {{if ne .ResourceRequirements "BestEffort"}}
```

Comment:
Seems a typo, should be ResourceRequirement, no s.

Reply:
thanks @haoqing0110  . typo fixed